samples/i2c/src/main.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2025 Space Cubics Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/i2c.h>
+
+#define TMP175_I2C_ADDR (0x4B)
+#define TMP175_TEMP_REG (0x00)
+
+int main(void)
+{
+	int ret;
+	uint8_t data[2];
+	float temp;
+
+	const struct device *i2c = DEVICE_DT_GET(DT_NODELABEL(i2c0));
+
+	if (!device_is_ready(i2c)) {
+		printf("I2C device is not ready\n");
+		ret = -1;
+		goto end;
+	}
+
+	ret = i2c_burst_read(i2c, TMP175_I2C_ADDR, TMP175_TEMP_REG, data, ARRAY_SIZE(data));
+	if (ret < 0) {
+		printf("Failed to read from Temperature Sensor (%d)\n", ret);
+		goto end;
+	}
+
+	data[1] = data[1] >> 4;
+	temp = (int8_t)data[0] + (float)data[1] * 0.0625f;
```

Comment:
What is ```0.0625f```?
We need to describe this I2C packet.

Reply:
I wrote the simple explanation in the user maulal.
https://github.com/spacecubics/scobc-a1-software-manual/blob/main/modules/ROOT/pages/use_driver.adoc#explain-the-sample-code-1

But will add the comment too.